data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/SimpleCrac.java

```diff
@@ -41,52 +40,27 @@
     private static final String SAME_ELEMENT_ID_DIFFERENT_NAME_ERROR_MESSAGE = "A network element with the same ID (%s) but a different name already exists.";
     private static final String SAME_CONTINGENCY_ID_DIFFERENT_ELEMENTS_ERROR_MESSAGE = "A contingency with the same ID (%s) but a different network elements already exists.";
 
-    private Map<String, NetworkElement> networkElements;
-    private Map<String, Instant> instants;
-    private Map<String, Contingency> contingencies;
-    private Map<String, State> states;
-    private Map<String, BranchCnec> branchCnecs;
-    private Map<String, RangeAction> rangeActions;
-    private Map<String, NetworkAction> networkActions;
-    private boolean isSynchronized;
-    private DateTime networkDate;
-
-    @JsonCreator
-    public SimpleCrac(@JsonProperty("id") String id,
-                       @JsonProperty("name") String name,
-                       @JsonProperty("networkElements") Set<NetworkElement> networkElements,
-                       @JsonProperty("instants") Set<Instant> instants,
-                       @JsonProperty("contingencies") Set<Contingency> contingencies,
-                       @JsonProperty("states") Set<State> states,
-                       @JsonProperty("cnecs") Set<BranchCnec> branchCnecs,
-                       @JsonProperty("rangeActions") Set<RangeAction> rangeActions,
-                       @JsonProperty("networkActions") Set<NetworkAction> networkActions) {
+    private final List<Instant> sortedInstants;
+    private final Map<String, NetworkElement> networkElements = new HashMap<>();
+    private final Map<String, Contingency> contingencies = new HashMap<>();
+    private final Map<String, State> states = new HashMap<>();
+    private final Map<String, BranchCnec> branchCnecs = new HashMap<>();
+    private final Map<String, RangeAction> rangeActions = new HashMap<>();
+    private final Map<String, NetworkAction> networkActions = new HashMap<>();
+    private boolean isSynchronized = false;
+    private DateTime networkDate = null;
+
+    public SimpleCrac(String id, String name, Set<Instant> instants) {
```

Comment:
rename argument instants to postContingencyInstants?
to avoid ambiguity in a case like: the user tries to create a crac with only outage and curative instants, and the simple crac automatically adds the preventive instant